examples/platforms/simulation/alarm.c

```diff
@@ -310,11 +310,9 @@ uint64_t otPlatTimeGet(void)
     return platformGetNow();
 }
 
-#if OPENTHREAD_CONFIG_TIME_SYNC_ENABLE
 uint16_t otPlatTimeGetXtalAccuracy(void)
```

Comment:
Implement a OT_WEAK function as default implementation for platforms not supporting this?

Reply:
I think it would be fine to add one to `radio_platform.cpp`. But currently it's not used anywhere, leading to a decrease in code coverage. I prefer to add it when we implemented the accuracy function. Thoughts?

Reply:
Sounds good.